_posts/2022-02-01-OLS-4-cohort-experience.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "OLS-4 cohort experience"
+image: https://github.com/open-life-science/open-life-science.github.io/blob/main/images/logo.png
+authors:
+   - Arent Kievits
+   - Open Science Community Delft
+---
+
+*A graduate of the fourth cohort of the Open Life Science (OLS) program talks about his experience!*
+
+As a PhD student at Delft University of Technology, I participated in the fourth cohort (named *Kenaz*) of the Open Life Science (OLS) program. OLS is a 16-week program where participants receive personal mentoring and training through cohort-calls. The main objectives are to:
```

Comment:
Add a link to OLS's (https://openlifesci.org) in case a reader is interested finding out more about it? :)